backend/src/main/scala/cromwell/backend/Backend.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.backend
+
+import cromwell.backend.model.{WorkflowDescriptor, JobDescriptor}
+
+/**
+  * Defines basic structure and functionality to work with a backend.
+  */
+trait Backend {
```

Comment:
I still don't quite get what the point of "Backend" is compared to BackendActor. Especially since this isn't on the diagram.